src/test/java/games/strategy/engine/lobby/server/login/LobbyLoginValidatorTest.java

```diff
@@ -41,6 +41,7 @@
 import games.strategy.security.TestSecurityUtils;
 import games.strategy.util.Tuple;
 
+@ExtendWith(MockitoExtension.class)
```

Comment:
@DanVanAtta said:
> You're right on this @ssoloff . I'm a bit curious if there is a JSR for extensions to be passed to nested classes. Another way to aks that is whether this is something IDEA should support, or if Eclipse is doing something non-standard

@DanVanAtta At your convenience, could you please try checking out this branch to see if this still an issue in IntelliJ?  As I theorized above, `MockitoExtension` and `Nested` tests in JUnit 5 are totally different than the `MockitoJUnitRunner` and `Enclosed` test runners in JUnit 4.  So there's reason to believe you may get a different result than before.

(BTW, I'm inclined to think IntelliJ was the outlier here as both Eclipse and Gradle supported the top-level-only-annotation style we tried to employ in the past.)

Reply:
> (BTW, I'm inclined to think IntelliJ was the outlier here as both Eclipse and Gradle supported the top-level-only-annotation style we tried to employ in the past.)

They may be inclined to fix it, one benefit of having revenue stream and payed employees : )

But more to the point, I did check out the branch, if I move the annotations to parent clases then these inner classes get marked as unused and are no longer run.

Reply:
Sorry, somehow I missed this comment.  So I need to revert d75e3ddec?  Right.